src/app/api/services/dot-workflow/dot-workflow.service.spec.ts

```diff
@@ -97,4 +97,13 @@ describe('DotWorkflowService', () => {
         ]);
         expect(this.lastConnection.request.url).toContain(`v1/workflow/contentlet/${inode}/actions`);
     });
+
+    it('should fire workflows page action', () => {
+        const inode = 'cc2cdf9c-a20d-4862-9454-2a76c1132123';
+        const actionId = '44d4d4cd-c812-49db-adb1-1030be73e69a';
+        spyOn(this.dotWorkflowService, 'fireWorkflowAction');
+        this.dotWorkflowService.fireWorkflowAction(inode, actionId);
```

Comment:
This test is not testing right... you are calling the method and then check if it was called and that's not what we need to test, because if I go to the service and change the url the request will failed but the test won't, what you need to test is the endpoint was called correctly, check the url and the method, ex:

https://github.com/dotCMS/core-web/blob/issue-13774-login-as-iframes/src/app/portlets/content-types/fields/service/field.service.spec.ts#L121

https://github.com/dotCMS/core-web/blob/issue-13774-login-as-iframes/src/app/api/services/paginator/paginator.service.spec.ts#L101